lib/travis/build/addons/coverity_scan.rb

```diff
@@ -51,7 +51,7 @@ def authorize_quota
   else
     WHEN=`echo $AUTH_RES | ruby -e "require 'rubygems'; require 'json'; puts JSON[STDIN.read]['next_upload_permitted_at']"`
     echo -e "\033[33;1mCoverity Scan analysis NOT authorized until $WHEN.\033[0m"
-    exit 1
+    exit 0
```

Comment:
Since we are exiting with `0` in both cases of the `if` statement, we should remove that logic from `src`, and add `assert: false` to `sh.raw` below.

This might require changes in the specs (untested).